src/itdelatrisu/opsu/states/MainMenu.java

```diff
@@ -720,6 +783,15 @@ public void mousePressed(int button, int x, int y) {
 			return;
 		}
 
+		// options button actions
+		if (selectOptionsButton.contains(x, y)) {
```

Comment:
put this in `mouseClicked` to solve the disappearing problem.
You'll also need to add 
```
if (showOptionsOverlay || !optionsOverlayProgress.isFinished()) return;
```
at the top of `mouseClicked`